apx: add page (#9501)
tldr-pages/tldr: pages/linux/apx.md

# apx

> Package management utility for Vanilla OS.

> > It installs packages inside managed containers or directly inside the host.

Author: ```suggestion
> Install packages inside managed containers or directly inside the host.
```
Would `Install` packages be better here?

> More information: <https://github.com/Vanilla-OS/apx>.

- Install a package in the system and initialize the container:

`sudo apx install --sys {{package}} && apx init`

- Install package(s) in the system or install AUR package(s) inside a container:

> `sudo apx install {{--sys|--aur}} {{package1 package2 ...}}`

Reviewer: We use this format
```suggestion
`sudo apx install --{{sys|aur}} {{package1 package2 ...}}`
```

> - Install a package from AUR (Use `apx --aur run {{package}}` , `apx --aur remove {{package}}` to run or remove an installed package):

Author: IMO, we can leave the extra commands here instead of creating a separate page. What do you guys think?

Reviewer 2: I suggest use something like `sudo apx {{--sys|--aur}} install {{package}}` to include sys and aur in one example.

`apx --aur run {{package}}`

- Update the list of available packages in the system:

`sudo apx --sys update`

- Upgrade all installed packages in the system to their newest available version:

`sudo apx --sys upgrade`

- Remove package(s) in the system or from the AUR container:

> `sudo apx {{--sys|--aur}} remove {{package1 package2 ...}}`

Author: ```suggestion
`sudo apx --{{sys|aur}} remove {{package1 package2 ...}}`
```
